frontend/src/App.vue

```diff
@@ -246,14 +181,6 @@ export default class App extends Mixins(UserInfoApiRequestMixin) {
       });
     }
   }
-
-  private goToRoute(path: string): void {
-    this.$router.push({ path });
-  }
-
-  private currentRouteHasTag(tag: RouteTag): boolean {
-    return this.$router.currentRoute.meta?.tag === tag;
```

Comment:
RouteTag importiert (Zeile 127) aber nicht mehr verwendet.  Siehe auch im workflow https://github.com/it-at-m/isi-frontend/actions/runs/6144120117/job/16668898929?pr=97